config/routes.rb

```diff
@@ -24,7 +24,7 @@
 
   get 'welcome/about'
   get 'welcome/lists'
-  #root 'todo_lists#index'
   root 'welcome#index'
+  get 'houses/index'
```

Comment:
This should produce a URL of `/houses`, but instead, it is producing a URL of `/houses/index`

you'll want to instead follow this pattern to achieve that:  https://github.com/hpjaj/things-my-kids-said/blob/master/config/routes.rb#L43